.github/workflows/coverity_scan.yml

```diff
@@ -0,0 +1,60 @@
+name: coverity-scan
+on:
+  workflow_dispatch: # because sometimes you just want to force a branch to have tests run
+  push:
+    branches:
+      - "**"
+    paths:
+      - "backend/**"
+      - .github/workflows/coverity_scan.yml
+  schedule:
+    - cron: '0 18 * * *' # Daily at 18:00 UTC
+
+defaults:
+  run:
+    working-directory: ./backend
+
+jobs:
+  latest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+      - name: Install dependencies
+        run: pushd ../frontend/ && yarn install && popd
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache Java Dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: gradle-build-${{ hashFiles('*.gradle', 'gradle/dependency-locks/*') }}
+      - name: Download Coverity Build Tool
+        run: |
+          wget -q https://scan.coverity.com/download/java/linux64 --post-data "token=$TOKEN&project=CDCgov/prime-data-input-client" -O cov-analysis-linux64.tar.gz
+          mkdir cov-analysis-linux64
+          tar xzf cov-analysis-linux64.tar.gz --strip 1 -C cov-analysis-linux64
+        env:
+          TOKEN: ${{ secrets.COVERITY_TOKEN }}
+      - name: Build with cov-build
+        run: |
+          export PATH=`pwd`/cov-analysis-linux64/bin:$PATH
+          cov-build --dir cov-int --fs-capture-search ../frontend ./gradlew clean assemble
+      - name: Submit the result to Coverity Scan
+        run: |
+          tar czvf sr.tgz cov-int
+          curl \
+            --form token=$TOKEN \
+            --form email=nicholas.a.robison@omb.eop.gov \
+            --form file=@sr.tgz \
+            --form version=trunk \
+            --form description="`git rev-parse --short HEAD`" \
+            https://scan.coverity.com/builds?project=CDCgov%2Fprime-data-input-client
```

Comment:
We're about to rename the project, is that going to cause pain and suffering?

Reply:
When you say ‘about’ does that mean hours, days or weeks? Renames are fairly simple.

Reply:
We've been talking about it for a week but didn't want to muck around while people were shoving multi-site out the door—I could do it now if that'd simplify life

Reply:
I think we should merge this now, and make the change when we're ready.

Reply:
The repo rename has happened, FWIW.

Reply:
Ah. Maybe I should read things more closely.

Reply:
Fixed. Would love to get this merged if you can approve.